pkg/task/cluster.go

```diff
@@ -41,12 +62,12 @@ func CreateKubeconfigFile(directory string, resource *model.ClusterDetails) erro
 	}
 
 	cluster := &clientcmdapi.Cluster{
-		Server:                resource.URL,
+		Server:                resource.Connection.Server,
 		InsecureSkipTLSVerify: resource.Insecure,
 	}
 
-	if resource.CAData != "" {
-		ca, err := base64.StdEncoding.DecodeString(resource.CAData)
+	if resource.Connection.CertificateAuthority != "" {
+		ca, err := base64.StdEncoding.DecodeString(resource.Connection.CertificateAuthority)
```

Comment:
This decoding needs to happen at the conversion time to the connection above. User input into the connection form won't be base64-encoded.

Reply:
Ah yes! Thanks for this.